### ISSUE TITLE
Calling collect on a CSV.File results in strange behavior

### ISSUE THREAD

**Issue by fredcallaway** [JuliaData/CSV.jl#391]
When you call collect on a `CSV.File`, it seems to succeed, giving you a list of `CSV.Row`s. And you can even access a column for one of those rows... one time. After that, you get missing everywhere.

```
csv = CSV.File("my.csv")
rows = collect(csv)
rows[1].col1  # a real value
rows[1].col1  # missing
```

**Comment by fredcallaway** [comment]
Note that you get this behavior in other situations, for example if you try to broadcast over a `CSV.File`. I think this could be the root cause for #387 as well.

**Comment by quinnj** [comment]
This will be fixed in #395

**Comment by quinnj** [comment]
Fixed in #395
